mongodb/src/main/java/com/yahoo/ycsb/db/AsyncMongoDbClient.java

```diff
@@ -0,0 +1,477 @@
+/**                                                                                                                                                                                
+ * Copyright (c) 2013-2014, Allanbank Consulting, Inc. All rights reserved.                                                                                                                             
```

Comment:
This will take some time for me to review how our licensing ought to work. Generally, for the contributions I've already seen to the project, the practice is for contributors to assign copyrights to Yahoo! (as the project originator) and then use an ASL license.